### PR TITLE
chore(dotnet): render enum name instead of literals

### PR DESCRIPTION
**Pull request by avodovnik**
Closes https://github.com/microsoft/playwright/issues/8497

### CODE REVIEW

**Comment by avodovnik** [comment]
Closes https://github.com/microsoft/playwright/issues/8497

File: dotnet/docs/api/class-browser.mdx [hunk 1]
```diff
@@ -105,7 +105,7 @@ Indicates that the browser is connected.
...
     - `Width` &#60;[int]&#62; Video frame width.
     - `Height` &#60;[int]&#62; Video frame height.
-  - `ReducedMotion`<a aria-hidden="true" tabindex="-1" class="list-anchor-link" id="browser-new-context-option-reduced-motion"/> &#60;"reduce"|"no-preference"&#62; Emulates `'prefers-reduced-motion'` media feature, supported values are `'reduce'`, `'no-preference'`. See [Page.EmulateMediaAsync(options)](./api/class-page.mdx#page-emulate-media) for more details. Defaults to `'no-preference'`.<a href="#browser-new-context-option-reduced-motion" class="list-anchor">#</a>
+  - `ReducedMotion`<a aria-hidden="true" tabindex="-1" class="list-anchor-link" id="browser-new-context-option-reduced-motion"/> &#60;`enum ReducedMotion { Reduce, No-preference }?`&#62; Emulates `'prefers-reduced-motion'` media feature, supported values are `'reduce'`, `'no-preference'`. See [Page.EmulateMediaAsync(options)](./api/class-page.mdx#page-emulate-media) for more details. Defaults to `'no-preference'`.<a href="#browser-new-context-option-reduced-motion" class="list-anchor">#</a>
```
Comment (yury-s) [inline dotnet/docs/api/class-browser.mdx:108]: `No-preference` => `NoPreference` ?

File: dotnet/docs/api/class-browser.mdx [hunk 2]
```diff
@@ -138,7 +138,7 @@ await page.GotoAsync("https://www.bing.com");
...
-  - `ColorScheme`<a aria-hidden="true" tabindex="-1" class="list-anchor-link" id="browser-new-page-option-color-scheme"/> &#60;"light"|"dark"|"no-preference"&#62; Emulates `'prefers-colors-scheme'` media feature, supported values are `'light'`, `'dark'`, `'no-preference'`. See [Page.EmulateMediaAsync(options)](./api/class-page.mdx#page-emulate-media) for more details. Defaults to `'light'`.<a href="#browser-new-page-option-color-scheme" class="list-anchor">#</a>
+  - `ColorScheme`<a aria-hidden="true" tabindex="-1" class="list-anchor-link" id="browser-new-page-option-color-scheme"/> &#60;`enum ColorScheme { Light, Dark, No-preference }?`&#62; Emulates `'prefers-colors-scheme'` media feature, supported values are `'light'`, `'dark'`, `'no-preference'`. See [Page.EmulateMediaAsync(options)](./api/class-page.mdx#page-emulate-media) for more details. Defaults to `'light'`.<a href="#browser-new-page-option-color-scheme" class="list-anchor">#</a>
```
Comment (yury-s) [inline dotnet/docs/api/class-browser.mdx:141]: ditto

File: dotnet/docs/api/class-browsertype.mdx [hunk 3]
```diff
@@ -101,7 +101,7 @@ var browser = await playwright.Chromium.LaunchAsync(new BrowserTypeLaunchOptions
...
-  - `ColorScheme`<a aria-hidden="true" tabindex="-1" class="list-anchor-link" id="browser-type-launch-persistent-context-option-color-scheme"/> &#60;"light"|"dark"|"no-preference"&#62; Emulates `'prefers-colors-scheme'` media feature, supported values are `'light'`, `'dark'`, `'no-preference'`. See [Page.EmulateMediaAsync(options)](./api/class-page.mdx#page-emulate-media) for more details. Defaults to `'light'`.<a href="#browser-type-launch-persistent-context-option-color-scheme" class="list-anchor">#</a>
+  - `ColorScheme`<a aria-hidden="true" tabindex="-1" class="list-anchor-link" id="browser-type-launch-persistent-context-option-color-scheme"/> &#60;`enum ColorScheme { Light, Dark, No-preference }?`&#62; Emulates `'prefers-colors-scheme'` media feature, supported values are `'light'`, `'dark'`, `'no-preference'`. See [Page.EmulateMediaAsync(options)](./api/class-page.mdx#page-emulate-media) for more details. Defaults to `'light'`.<a href="#browser-type-launch-persistent-context-option-color-scheme" class="list-anchor">#</a>
```
Comment (yury-s) [inline dotnet/docs/api/class-browsertype.mdx:104]: ditto

File: src/generator.js [hunk 4]
```diff
@@ -388,16 +388,21 @@ import TabItem from '@theme/TabItem';`);
    * @param {'in'|'out'} direction
    * @param {Documentation.Member} member
    */
-  renderType(type, direction, member) {
+   renderType(type, direction, member) {
```
Comment (yury-s) [inline src/generator.js:391]: revert?

**Review by yury-s — APPROVED** [review]

File: src/generator.js (hunk 4 above)
Comment (avodovnik) [inline src/generator.js]:
```suggestion
  renderType(type, direction, member) {
```

**Pull request merged**